prime-router/metadata/schemas/AZ/az-covid-19-csv.schema

```diff
@@ -2,67 +2,65 @@
 name: az-covid-19
 description: AZ COVID-19 flat file
 topic: covid-19
-trackingElement: standard.testing_lab_specimen_id
+trackingElement: testing_lab_specimen_id
+basedOn: covid-19
 elements:
-  - name: standard.sending_application
-    type: TEXT
+  - name: sending_application
     csvFields: 
       - name: Sending_Application
+        format: $name
+      - name: Sending_Application_ID
+        format: $universalId
 
-  - name: standard.sending_application_id
-    type: HD
-    csvFields:
-    - name: Sending_Application_ID
-
-  - name: standard.testing_lab_name
+  - name: testing_lab_name
     csvFields:
     - name: Lab_Name
 
-  - name: standard.testing_lab_clia
+  - name: testing_lab_clia
```

Comment:
A thought:   AZ expects fewer columns than are incoming from SimpleReport.   So there does need to be a way to say "out of all the covid-19 fields, please only include these...testing_lab_name, testing_lab_client, etc ,etc"

But in most cases, all the other metadata about those fields is identical to the parent schema.   Is there a way we can have a syntax that consolidated this long file, and simply lists the fields we want to display, in the most common cases where we're re-using the csvField column name etc.   Example:

```
  name: az-covid-19
  description: AZ COVID-19 flat file
  reuse-from-parent-schema: { testing_lab_name, testing_lab_client, etc ,etc }
```

With that list at the top of the file, then the rest of az-covid-19-csv.schema is simply a list of those places where we are overriding the metadata in the parent schema, eg, to use an $alt schema, or a different csvField columnName.   Much more readable.